client/requestUtil.js

```diff
@@ -15,6 +15,9 @@ const EXPIRED_CREDENTIAL_ERRORS = [
   /The provided token has expired\./,
   /Invalid according to Policy: Policy expired\./
 ]
+const SQS_MAX_LIST_MESSAGES_COUNT = 10
```

Comment:
can you move this into `./config` for cleanliness

Reply:
yep sure, but do you think that I should move the upper block there as well?
const PUT_CONCURRENCY = 100
const S3_MAX_RETRIES = 1
const EXPIRED_CREDENTIAL_ERRORS = [
  /The provided token has expired\./,
  /Invalid according to Policy: Policy expired\./
]

Reply:
I think that is also a good idea